applications/coreutils/mkdir.cpp

```diff
@@ -41,17 +60,17 @@ void mkdir_parent_dirs(const char *path)
 }
 
 int main(int argc, char **argv)
-{
+{    
     if (argc == 2)
     {
         return mkdir(argv[1]);
     }
 
-    if (argc == 3)
+    argc = cmdline_parse(&cmdline, argc, argv);
```

Comment:
cmdline_parse should be called before everything because he will consume some argv values?

Reply:
How should I check for the case `mkdir folderpath` using cmdline_parse?

Reply:
cmdline_parse will remove all option like -p and won't touch the rest ;)

Reply:
applications/coreutils/ls.cpp is a good exemple ;)